reference/6/Microsoft.PowerShell.Management/Get-Content.md

```diff
@@ -401,30 +438,57 @@ Accept wildcard characters: False
-- **Default**: Encodes using the default value: ASCII.
-- **OEM**: Uses the default encoding for MS-DOS and console programs.
-- **Byte**: Encodes a set of characters into a sequence of bytes.
-- **String**: Uses the encoding type for a string.
+- **OEM**: Uses UTF8NoBOM on non-Windows and the encoding for MS-DOS and console programs on Windows.
+- **String**: Uses Unicode encoding.
 - **Unicode**: Encodes in UTF-16 format using the little-endian byte order.
 - **UTF7**: Encodes in UTF-7 format.
-- **UTF8**: Encodes in UTF-8 format.
+- **UTF8**: Encodes in UTF-8 format without Byte Order Mark (BOM).
 - **UTF8BOM**: Encodes in UTF-8 format with Byte Order Mark (BOM)
-- **UF8NOBOM**: Encodes in UTF-8 format without Byte Order Mark (BOM)
+- **UF8NoBOM**: Encodes in UTF-8 format without Byte Order Mark (BOM)
 - **UTF32**:  Encodes in UTF-32 format.
-- **Unknown**: The encoding type is unknown or invalid; the data can be treated as binary.
+- **Unknown**: The encoding type is unknown, but is treated as Unicode.
 
-Encoding is a dynamic parameter that the FileSystem provider adds to the `Get-Content` cmdlet. This parameter works only in file system drives.
+Encoding is a dynamic parameter that the FileSystem provider adds to the `Get-Content` cmdlet.
+This parameter is available only in file system drives.
 
-When reading from and writing to binary files, use a value of Byte for the Encoding dynamic parameter and a value of 0 for the ReadCount parameter.  A ReadCount value of 0 reads the entire file in a single read operation and converts it into a single object (PSObject).  The default ReadCount value, 1, reads one byte in each read operation and converts each byte into a separate object, which causes errors when you use the `Set-Content` cmdlet to write the bytes to a file. For more information, see the examples.
+When reading from and writing to binary files, use the `-AsByteStream` parameter and
+a value of 0 for the ReadCount parameter.
+A ReadCount value of 0 reads the entire file in a single read operation.
+The default ReadCount value, 1, reads one byte in each read operation and converts each byte into a
+separate object, which causes errors when you use the `Set-Content` cmdlet to write the bytes
+to a file unless you use `-AsByteStream` parameter.
+For more information, see the examples.
 
 > [!NOTE]
-> Beginning in PowerShell 6.0, **Byte** is no longer a valid option for the `-Encoding` parameter. You can use the `-AsByteStream` parameter to indicate that the content should be read and output as a byte stream.
+> Beginning in PowerShell 6.0, the type of the Encoding parameter changed from
+> `FileSystemCmdletProviderEncoding` to `System.Text.Encoding`, and **Byte** is no longer
+> a valid option for the `-Encoding` parameter.
+> You can use the `-AsByteStream` parameter to indicate that the content should be read and
+> output as a byte stream.
+
+- PowerShell 6
+
+```yaml
+Type: System.Text.Encoding
```

Comment:
Having two yaml blocks is not allowed by PlatyPS. You should only document the v6.x information in this v6 document.

Reply:
will do